templates/docs/examples/patterns/search-box/_script.js

```diff
@@ -0,0 +1,15 @@
+function initSearchResetButtons(selector) {
+  var resetButtons = [].slice.call(document.querySelectorAll(selector));
+
+  resetButtons.forEach(function (button) {
+    button.addEventListener('click', function () {
+      var input = button.parentNode.querySelector('input');
+
+      input.focus();
+    });
+  });
+}
+
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
While it's not a bad idea, I don't think we use DOMContentLoaded in any other example.